e2e/exec/exec_test.go

```diff
@@ -276,7 +276,7 @@ var _ = Describe("exec flow", func() {
 					EnvName: envName,
 				})
 				return resp, err
-			}, "60s", "10s").ShouldNot(BeEmpty())
+			}, "60s", "20s").ShouldNot(BeEmpty())
```

Comment:
Should we be increasing the duration from `60s` to `120s` or something? 

I thought this syntax meant "check every 10s if we get a non-nil response up to 60s